htdocs/contact/class/contact.class.php

```diff
@@ -495,11 +497,13 @@ function update_perso($id, $user=0)
 	 *  Load object contact
 	 *
 	 *  @param      int		$id          id du contact
+	 *  @param	string		$ref_ext	External reference, not given by Dolibarr
 	 *  @param      User	$user        Utilisateur (abonnes aux alertes) qui veut les alertes de ce contact
 	 *  @return     int     		    -1 if KO, 0 if OK but not found, 1 if OK
 	 */
-	function fetch($id, $user=0)
+	function fetch($id, $ref_ext, $user=0)
```

Comment:
You're absolutely write, I noticed that after sending the PR, will send a new one soon.